src/Microsoft.Kusto.ServiceLayer/Admin/AdminService.cs

```diff
@@ -64,15 +56,15 @@ public static AdminService Instance
         /// <summary>
         /// Initializes the service instance
         /// </summary>
-        public void InitializeService(ServiceHost serviceHost)
+        public void InitializeService(IProtocolEndpoint serviceHost)
```

Comment:
Replaced by a completely different interface. Is this expected?

Reply:
The ServiceHost implements the IProtocolEndpoint. I just changed the param to it's interface instead of the ServiceHost. I no longer need the change so I'll revert it.